src/modules/setup/BuilderHubDatasource.ts

```diff
@@ -0,0 +1,70 @@
+import * as retry from 'async-retry'
+import axios from 'axios'
+import { getAccount, getToken, getWorkspace } from '../../conf'
+import { region } from '../../env'
+import log from '../../logger'
+
+const SECOND = 1000
+const builderHubTimeout = 2 * SECOND
+
+const retryOpts = {
+  retries: 2,
+  minTimeout: 1000,
+  factor: 2,
+}
+
+export class BuilderHubDatasource {
```

Comment:
I think this should be only `BuilderHub` and you could move it to `src/clients`.